site/pages/react-native/signer/authenticating-users.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Authenticating Users with the React Native Signer
+description: Learn how to authenticate users with the React Native Signer
+---
+
+# Authenticating Users
+
+This guide assumes you have already followed the [Setup Guide](/react-native/signer/setup-guide) and have a basic understanding of how to use the Signer in your project.
+
+## Create a Signer Instance
+
+```tsx
+import { RNAlchemySigner } from "@account-kit/react-native-signer";
+
+const signer = new RNAlchemySigner({
+  client: { connection: { apiKey: API_KEY } },
+});
+```
+
+## Authenticate a User
+
+Next, you'll need to authenticate your user before you can use the Signer as an owner on the account
+
+:::info
+At the time of writing, the React Native Signer only supports Email at the moment.
+:::
+
+### Send an Email Magic Link to a User
+
+To send an email magic link to a user, you can use the `signer.authenticate()` method.
+
+```tsx
+signer
+  .authenticate({ email: "user@example.com", type: "email" })
+  .catch((error) => {
+    console.error(error);
+  });
+```
+
+### Authenticate User via Deep Link
+
+When a user clicks on the magic link in their email, it should deep link to your app if this has been setup correctly.
+
+A `bundle` parameter present in the deep link url will be used to authenticate the user and save the user's session.
+
+Here's an example of what this might look like:
+
+:::code-group
+
+```tsx [example.tsx]
```

Comment:
nit: an we add 
```suggestion
```tsx [example.tsx] twoslash
```

to this to make sure things are actually correct? if it's not, it will cause build errors so we have more typesafety in our docs that way